src/diffusers/pipelines/ddim/pipeline_ddim.py

```diff
@@ -68,6 +69,9 @@ def __call__(
                 [PIL](https://pillow.readthedocs.io/en/stable/): `PIL.Image.Image` or `np.array`.
             return_dict (`bool`, *optional*, defaults to `True`):
                 Whether or not to return a [`~pipeline_utils.ImagePipelineOutput`] instead of a plain tuple.
+            substeps_mode (`str`, *optional*, defaults to "linear"):
```

Comment:
Hmm this is a bit too hacky for me and not really related to DDIM anymore IMO. DDIM is defined as selecting substeps linearly IMO. Do we really need this?

Reply:
We probably don't need this, but in the [DDIM paper](https://arxiv.org/pdf/2010.02502.pdf), they explored a non-linear option (see Appendix D.2 on p.18) and in principle you can select the steps however you want.
Should I remove this since it's not supported yet anyway?